src/main/java/com/github/tomakehurst/wiremock/core/WireMockConfiguration.java

```diff
@@ -122,6 +134,8 @@ public class WireMockConfiguration implements Options {
   private boolean stubCorsEnabled = false;
   private boolean disableStrictHttpHeaders;
 
+  private Boolean proxyPassThrough;
```

Comment:
Can we change this to `boolean`?

Reply:
If I change this to primitive type then it will always take the default value as false so I feel here it is not required. Let me know if you feel otherwise.

Reply:
I am assigning this value as true as I can see done for others and incorporating this change as well.